algebird-core/src/main/scala/com/twitter/algebird/Aggregator.scala

```diff
@@ -495,7 +495,7 @@ trait Aggregator[-A, B, +C] extends java.io.Serializable { self =>
     new MonoidAggregator[A, Option[B], Option[C]] {
       override def prepare(input: A): Option[B] = Some(self.prepare(input))
       override def present(reduction: Option[B]): Option[C] = reduction.map(self.present)
-      override def monoid = new OptionMonoid[B]()(self.semigroup)
+      override def monoid: OptionMonoid[B] = new OptionMonoid[B]()(self.semigroup)
```

Comment:
can we relax this to `Monoid[Option[B]]`? I hate to make the types needlessly tight.